lms/templates/courseware/course_about.html

```diff
@@ -194,9 +195,11 @@
               href_class = "register"
           %>
           <div class="inquire-wrapper">
+            % if course.advertised_start is None:
```

Comment:
@haikuginger I'm confused about what to do when „advertised start date“ is "", that is, an empty string (but not null). CMS differentiates between null and "", and my concern is that strange things happen when it's "": the „Enroll now“ is NOT be visible, and the date shown is not "" (empty string) as mandated but is the course start date, this seems inconsistent (because if the course shows a start date instead of a text, then it should be enrollable).
CMS says that null == „the start date that you have set for this course is used.“, and I think we should make "" operate in the same way as null. To do that we need to show the „enroll button“ also when "".
I have tried how to detect "" and found that this line would work better (i.e. same for null/""):

            % if course.advertised_start is None or course.advertised_start == "":

But maybe we want to leave the "" case undefined or discouraged it.

Reply:
@clemente, I think you're right; I can just check for truthiness by doing the following:

```
    % if course.advertised_start:
```